mod/05_email/set/right/bcc.rb

```diff
@@ -18,7 +22,7 @@ def process_email_addresses context_card, format_args, args
 format :email_text do
   view :email_addresses do |args|
     context = args[:context] || self
-    card.item_names.map do |item_name|
+    card.item_names(:context=>context.cardname).map do |item_name|
```

Comment:
I looked into this.  item_names absolutizes already (unless you use :context=>:raw), so it looks like we can drop the next line.